typescript/looker/rtl/transport.ts

```diff
@@ -188,52 +214,113 @@ export interface ITransport {
 }
 
 /** A successful SDK call. */
-interface ISDKSuccessResponse<T> {
+export interface ISDKSuccessResponse<T> {
   /** Whether the SDK call was successful. */
-  ok: true;
+  ok: true
   /** The object returned by the SDK call. */
-  value: T;
+  value: T
 }
 
 /** An erroring SDK call. */
-interface ISDKErrorResponse<T> {
+export interface ISDKErrorResponse<T> {
   /** Whether the SDK call was successful. */
-  ok: false;
+  ok: false
   /** The error object returned by the SDK call. */
-  error: T;
+  error: T
 }
 
 /** An error representing an issue in the SDK, like a network or parsing error. */
 export interface ISDKError {
-  type: 'sdk_error';
-  message: string;
+  type: 'sdk_error'
+  message: string
 }
 
 export type SDKResponse<TSuccess, TError> =
   | ISDKSuccessResponse<TSuccess>
   | ISDKErrorResponse<TError | ISDKError>
 
 /**
- * Base authorization interface
+ * Basic authorization session interface for most API authentication scenarios
  */
-export interface IAuthorizer {
-  settings: IApiSettings;
-  transport: ITransport;
-
-  /** is the current session authenticated? */
-  isAuthenticated(): boolean;
-
-  authenticate(init: IRequestInit): Promise<IRequestInit>;
+export interface IAuthSession  {
```

Comment:
it seems to me like `IAuthSession` logically belongs more in `authSession.ts`? or does that create a circular import issue?

Reply:
I don't think Typescript has circular import issues. Now that the base `AuthSession` is intended as the common parent for all auth session variants, moving that interface into the same file makes sense. Thanks.